src/main/groovy/io/seqera/wave/service/builder/DockerBuildStrategy.groovy

```diff
@@ -53,6 +53,9 @@ class DockerBuildStrategy extends BuildStrategy {
     @Value('${wave.build.singularity-image}')
     String singularityImage
 
+    @Value('${wave.build.singularity-image-arm}')
+    String singularityImageArm
```

Comment:
```suggestion
    @Value('${wave.build.singularity-image-arm64}')
    String singularityImageArm64
```

Let's rename to arm64 suffix for consistency. 

Also, this can be made nullable, and the default to `singularityImage + '-arm64'`

Reply:
done

Reply:
Maybe i'm wrong here, but it looks like that when `singularityImageArm64` is not specified `null` it will be used. 

My suggestion was instead that when `singularityImageArm64` is not specied (nullable) the value of `singularityImage` is appending to it the `-arm64`, suffix. 


To avoid duplicating the code, it could make sense to introduce an object `BuildConfig` (along the same way of SpackConfig) that holds all the `wave.build.*` settings

Reply:
But in this way, we are coupling our code with the naming convention of singularity docker image. If somewhere down the line they change this, then our code will be broken

Reply:
True, but still remain the ability to override it.

https://en.m.wikipedia.org/wiki/Convention_over_configuration#:~:text=Convention%20over%20configuration%20(also%20known,repeat%20yourself%20(DRY)%20principles.

Reply:
ok i understood now, thanks for the clarification

Reply:
done